main.go

```diff
@@ -36,9 +37,11 @@ func main() {
 
 	router := mux.NewRouter().StrictSlash(true)
 
+	router.HandleFunc("/mentor/oauth", controllers.MentorOAuth).Methods("POST")
```

Comment:
Let's divide the urls into subrouters and add only the subrouters in main.go. Subroutes can register urls in views. I will do that in another PR. It's fine for now.

Reply:
Acha cool !